arch/arm/src/stm32/Kconfig

```diff
@@ -6381,6 +6381,48 @@ config STM32_SPI_DMA
 	---help---
 		Use DMA to improve SPI transfer performance.  Cannot be used with STM32_SPI_INTERRUPT.
 
+config STM32_SPI1_DMA
```

Comment:
Do we want to have TX and RX granularity on the DMA? TX are usually tiny. The DMA setup would take longer then the TX to the bus.

Reply:
I was wondering about that, we'd have to further split spi_exchange.

![image](https://user-images.githubusercontent.com/84712/42481739-fa327390-83b1-11e8-9fd6-63c3d95beff8.png)

![image](https://user-images.githubusercontent.com/84712/42481745-0113a576-83b2-11e8-84cf-3803682db904.png)

Reply:
On second thought I don't think it makes any sense to split it because of the way SPI works we really 2 DMA per SPI